lib/src/cmaf/ham/mapper/hls/mapHamToHls.ts

```diff
@@ -1,45 +1,57 @@
 import { Manifest } from '../../types';
 import {
 	AudioTrack,
-	TextTrack,
 	Presentation,
+	SelectionSet,
+	SwitchingSet,
+	TextTrack,
+	Track,
 	VideoTrack,
 } from '../../types/model';
-import {
-	NEW_LINE,
-	WHITE_SPACE,
-	WHITE_SPACE_ENCODED,
-	AT_SEPARATOR,
-	HYPHEN_MINUS_SEPARATOR,
-} from '../../../utils/constants.js';
+import { getPlaylistData, getSegments } from './utilsHamToHls.js';
+
+type ManifestPlaylistPiece = {
+	manifestToConcat: string;
```

Comment:
We should find a better way to name these variables. It's weird to have `manifestToConcat` as part of a type. What I see, the `manifestToConcat` is the line to add in the main playlist, so maybe the name could be related to something like that.